packages/create-sitecore-jss/src/bin.ts

```diff
@@ -98,6 +113,23 @@ export const main = async (args: ParsedArgs) => {
     args.force = true;
   }
 
+  let preCommitHook = args.preCommitHook;
+  if (!preCommitHook) {
```

Comment:
If `preCommitHook` is a string, then we'd need to check for 'no' value here as well.